scripts/build.d/bzip2

```diff
@@ -2,11 +2,26 @@
 
 set -e
 
-pkgname=bzip2
-pkgbranch=${VERSION:-master}
-pkgfull=$pkgname-$pkgbranch
+if [ -z "${SYNCGIT}" ]; then
+  pkgname=bzip2
+  pkgver=${VERSION:-master}
```

Comment:
Please use a released version.  (I assume there is one.)

Reply:
The latest release version is [1.0.8](https://gitlab.com/bzip2/bzip2/-/tree/bzip2-1.0.8), but this version still using **Makefile** as build system also having problem with building **dynamic file** at MacOS, therefore I choose master as default version.

Reply:
What if we go with a static library?

Reply:
@j8xixo12 Any feedback on this?

Reply:
Sorry for the late reply, you meant solvcon using bzip2 with static library?
if yes maybe I can try it this weekends !

Reply:
This is not an urgent issue and not late at all.  Yes, something like that, but it is in `devenv`, we build bzip2 static lib instead of dynamic lib.  If the consumer of bzip2 accepts static lib this may be a workaround?

Reply:
Bzip2 also build static library at v.1.0.8 and solvcon can be build with bzip2 static library,  but when I ran the test, test failed due to ```ModuleNotFoundError: No module named '_bz2'```, it seems python can not find bzip2 with static library ?

Reply:
In theory it is possible to build Python using static `bzip2` but you would probably need to look into Python's make file to know how to configure it that way.